hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/evm/store/contract/MirrorEntityAccessTest.java

```diff
@@ -85,37 +86,23 @@ void isUsableWithPositiveBalance() {
         assertThat(result).isTrue();
     }
 
-    @Test
-    void isNotUsableWithNegativeBalance() {
-        final long balance = -1L;
-        when(store.getAccount(ADDRESS, OnMissing.DONT_THROW)).thenReturn(account);
-        when(account.getBalance()).thenReturn(balance);
-        final var result = mirrorEntityAccess.isUsable(ADDRESS);
-        assertThat(result).isFalse();
-    }
-
-    @Test
-    void isNotUsableWithWrongAlias() {
-        final var address = Address.fromHexString("0x3232134567785444e");
```

Comment:
This was wrong due to to bad entry. I had changed it to 
```suggestion
   final var address = Address.fromHexString("0x0");
```

Reply:
Accepted (but I did my own push of a new commit to _reinclude_ the test)